bin/kotlin-client-all.sh

```diff
@@ -8,3 +8,4 @@
 ./bin/kotlin-client-string.sh
 ./bin/kotlin-client-threetenbp.sh
 ./bin/kotlin-client-nullable.sh
+./bin/kotlin-client-retrofti2.sh
```

Comment:
typo: kotlin-client-retrofti2.sh => kotlin-client-retrofit2.sh

Reply:
Thanks for reviewing! 
It's solved in this commit: a3b34b5